src/user/supercritical_initialization.F90

```diff
@@ -32,6 +33,8 @@ subroutine supercritical_set_OBC_data(OBC, G, GV, US, param_file)
   ! Local variables
   character(len=40)  :: mdl = "supercritical_set_OBC_data" ! This subroutine's name.
   real :: zonal_flow ! Inflow speed [L T-1 ~> m s-1]
+  integer :: unrot_dir ! The unrotated direction of the segmetn
```

Comment:
should be spelled segment

Reply:
Thank you for catching this spelling error.  It has been corrected.